docs/nodes/README.md

```diff
@@ -15,7 +15,6 @@ This section will focus on how to operate full nodes, validators and light clien
 - [Validator Guides](./validators.md)
   - [Running in Production](./running-in-production.md)
   - [How to secure your keys](./validators.md#validator_keys)
-  - [Remote Signer](./remote-signer.md)
```

Comment:
If I understand correctly we're just removing the test harness, not the remote signing facility more generally. If that's right, should we leave the docs linked?

(The one in tools/README.md should probably go—but this one I think is not about validating the signer, but signing in the validator)

Reply:
yeah, sorry, overeager!

reverted this.